src/Features/CSharp/Portable/Completion/CompletionProviders/ImportCompletion/TypeImportCompletionProvider.cs

```diff
@@ -74,5 +76,24 @@ protected override bool IsFinalSemicolonOfUsingOrExtern(SyntaxNode directive, Sy
 
             return false;
         }
+
+        protected override HashSet<string> GetAliasSymbolsName(SyntaxNode node, SemanticModel semanticModel, CancellationToken cancellationToken)
+        {
+            var symbolNames = new HashSet<string>();
+            foreach (var candidateNode in node.GetEnclosingUsingDirectives())
+            {
+                if (candidateNode.Alias != null)
```

Comment:
The part that retrieve symbol/name is language agnostic, consider moving up to the base class.

Reply:
Done